projects/icu/project.yaml

```diff
@@ -13,6 +13,8 @@ auto_ccs:
  - elango@unicode.org
 sanitizers:
  - address
+ - undefined:
+    experimental: True 
```

Comment:
I wouldn't even bother making it experimental, you won't get the bugs filed and you will have to go to oss-fuzz.com yourself to see them. Still want to use it?